src/panels/lovelace/common/handle-click.ts

```diff
@@ -53,7 +54,9 @@ export const handleClick = (
 
   switch (actionConfig.action) {
     case "more-info":
-      if (config.entity || config.camera_image) {
+      if (actionConfig.card) {
+        showCardDialog(node, { card: actionConfig.card });
```

Comment:
I was thinking of using the actual `more-info-controls` and overruling the `more-info-content` with a `type` like we do for cards and badges.

We could later also add options to hide things like history or state.

Reply:
@bramkragten I think that history, state, and the default controls should be hidden by default, otherwise this won't look good.

Reply:
The default controls is `more-info-controls` and if you want to make a generic customisable `more-info`, the history and state should not be hidden by default.

I don't want this to be a `more-info` dialog for Lovelace cards, it should be a `more-info` dialog you can customize. Otherwise we will have all different kind of dialogs in the future and it would be a pain to maintain.

Secondly, if we show a more info, it should also behave as a more info.

Reply:
That would make this dependent on an entity which is should not be

Reply:
If someone wants to show history or state, they can do that with cards. Only thing missing would be attributes perhaps, an attributes card might be a nice addition though to Lovelace